### DISCUSSION TITLE
Images in RSS Feeds?

### DISCUSSION THREAD

**Discussion by Aasemoon** [bluesky-social/atproto#2543] [How to do it?]
I'm not sure if I should post this in the ideas category, or if the solution already exists, so I'll just put this here. Is there currently a way to see images and links inside the RSS feed items? I'd appreciate any hints.
Cheers!

**Comment by johnwittacre** [comment]
Would love this as well. I follow a lot of image accounts and it would be great to see the content right in the rss feed.
